### DISCUSSION TITLE
(solved) Adblockers: data-api attribute not being checked by script.

### DISCUSSION THREAD

**Discussion by verymilan** [plausible/analytics#1467] [Anything Else]
Hi there, i was just trying to get some more people tracked, following your manual about dealing with Adblockers. Unfortunately, `POST` requests are still being blocked as they are going to the original domain. At least in the `outbound-links.js`, i didn't spot a check for `data-api`, which might be the reason... .

**Comment by verymilan** [comment]
that was odd, maybe last time the plausible update didn't work or something? now it's there o.O

**Comment by metmarkosaric** [comment]
great, thanks for the update @verymilan! enjoy Plausible!
